docs/artifact-descriptions/vmcai26.md

```diff
@@ -0,0 +1,213 @@
+# VMCAI '26 Artifact Description
+## Data Race Detection by Digest-Driven Abstract Interpretation
+
+This is the artifact description for our [VMCAI '26 paper "Data Race Detection by Digest-Driven Abstract Interpretation](https://arxiv.org/abs/2511.11055).
+The artifact is available on [Zenodo](http://dx.doi.org/10.5281/zenodo.17128591).
+
+**The description here is provided for convenience and not maintained.**
+The artifact contains [Goblint at `race_digest_staging` branch](https://github.com/goblint/analyzer/tree/race_digest_staging).
```

Comment:
It could be this now, right?
```suggestion
The artifact contains [Goblint at `vmcai26` git tag](https://github.com/goblint/analyzer/releases/tag/vmcai26).
```